### PR TITLE
Seeds generate

### PR DESCRIPTION
**Pull request by albertoT75**
Please review

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
 # Examples:
-#
+puts "Destroy previous users"
+User.destroy_all
+puts "Destroy previous users"
+Pet.destroy_all
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+USER = User.create(username: "Leon", email: "leon@example.com", password: "123456")
+counter = 0
```
Comment (0math) [inline db/seeds.rb]: we don't need a counter here

File: db/seeds.rb [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+10.times do
+  counter += 1
```
Comment (0math) [inline db/seeds.rb]: also counter

File: db/seeds.rb [hunk 3]
```diff
@@ -2,6 +2,25 @@
...
+ Pet.create(
+    name: Faker::Name.name,
+    address: Faker::Address.full_address,
+    breed: Faker::Creature::Animal.name,
+    age: rand(1..20),
+    description: Faker::Quote.most_interesting_man_in_the_world,
+    user_id: USER.id )
+ puts "created pet number #{counter}"
```
Comment (0math) [inline db/seeds.rb]: and here, too

**Comment by albertoT75** [comment]
Tested ok

**Pull request merged**